account_asset_management/views/account_move.xml

```diff
@@ -4,7 +4,6 @@
         <field name="name">account.move.form.account.asset.management</field>
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
-        <field name="groups_id" eval="[(4, ref('account.group_account_invoice'))]" />
```

Comment:
The rest of the fields should have the group as well.

Reply:
i think its ok now

Reply:
done